tests/test_circulation_data.py

```diff
@@ -0,0 +1,265 @@
+from nose.tools import (
+    eq_,
+    set_trace,
+)
+
+from copy import deepcopy
+import datetime
+
+from metadata_layer import (
+    CirculationData,
+    ContributorData,
+    FormatData,
+    IdentifierData,
+    LinkData,
+    ReplacementPolicy,
+    SubjectData,
+)
+
+from model import (
+    DataSource,
+    DeliveryMechanism,
+    Hyperlink, 
+    Identifier,
+    Representation,
+    RightsStatus,
+    Subject,
+)
+
+from . import (
+    DatabaseTest,
+    DummyHTTPClient,
+)
+
+from s3 import DummyS3Uploader
+
+class TestCirculationData(DatabaseTest):
```

Comment:
it would be more consistent with the rest of the code base to put this class in test_metadata.py, since CirculationData is in metadata.py